config/config.go

```diff
@@ -10,42 +10,37 @@ import (
 	"github.com/mitchellh/go-homedir"
 )
 
-var (
-	config                Config // holds the global pick config.
+const (
 	defaultConfigFileTmpl = "%s/.pick/config.toml"
 )
 
-func Execute() error {
-	home, err := homedir.Dir()
-	if err != nil {
-		return err
-	}
-
-	configFile := fmt.Sprintf(defaultConfigFileTmpl, home)
-	if err := initConfig(configFile); err != nil {
-		return err
-	}
-
-	return RootCmd.Execute()
-}
-
 type Config struct {
 	Encryption crypto.Config
```

Comment:
We export `Encryption` and `Storage` because they're referenced when we load the safe inside the `commands` package.

There is no reason `Version` is exported.